README.rst

```diff
@@ -1,4 +1,4 @@
-.. Copyright 2020-2022 Robert Bosch Car Multimedia GmbH
```

Comment:
Hi Holger,
the "Car Multimedia" is in all files if I'm not wrong.
I saw this in all reviews. But I think it's not very important.
As per my understanding correct would be now "Robert Bosch GmbH".
Thank you,
 Thomas